src/main/java/io/fabric8/vertx/maven/plugin/mojos/AbstractRunMojo.java

```diff
@@ -182,31 +182,23 @@ protected void addVertxArgs(List<String> argsList) {
         argsList.add(vertxCommand);
```

Comment:
Some general comment with respect to 'filling in member variables' : In general I would try to avoid keeping state in the MoJo like its here with the member variable `argsList` (which is also passed around as argument to member methods, which would be unnecessary). This is not only problematic when this singleton is called in a concurrent manner, but also confusing for the reader.

In this concrete example I would create the `argsList` as a local variable and either path it around via arguments (as you do in this case) or even use it as an return value (e.g. `List<String> getVertxArgs()` instead of `void addVertxArgs(List<String> args)`. Using methods with side effects (i.e. all methods manipulating the arguments) are also hard to understand and debug. (That's one of the reasons why functional programming tries to avoid side effects at all costs).